robottelo/decorators.py

```diff
@@ -273,12 +274,18 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
+    upstream_mode = str(conf.properties.get('main.upstream', 1)) == '1'
```

Comment:
I used str() conversion because my test data was returning integer.  I agree with the comment.  I will fix the test data to sent str to mimic the real config behavior.